shared/people/invite-friends/invite-contacts.native.tsx

```diff
@@ -52,6 +52,29 @@ const InviteContacts = () => {
   const placeholderText = loading ? '' : `Search ${contacts.length.toLocaleString()} contacts`
 
   const disabled = !!successCount
+  const emailsDisabled = disabled || !!selectedPhones.size
+  const phonesDisabled = disabled || !!selectedEmails.size
+
+  const anyPhonesSelected = !!selectedPhones.size
+  const anyEmailsSelected = !!selectedEmails.size
+  const [anySelected, setAnySelected] = React.useState(false)
+  React.useEffect(() => {
+    const newAnySelected = anyPhonesSelected || anyEmailsSelected
+    if (newAnySelected !== anySelected) {
+      // this RAF is a hack so we don't animate the list items changing enabled <-> disabled
+      requestAnimationFrame(() => {
```

Comment:
how does this work?

Reply:
functionally here it delays setting the local `anySelected` state until after the the list's disabled states change. Otherwise the layoutanimation animated it all at once which didn't look good. Don't really know whether rAF is right for this though, it's delayed longer than I expected on my profile build